### ISSUE TITLE
Unknown options when working with 5eTools JSON

### ISSUE THREAD

**Issue by isthei** [ebullient/fc5-convert-cli#57]
Hey, I'm a DM with only basic experience with Java trying to convert 200+ 5eTools homebrew monsters into a Fight Club 5 XML. From what I understand, the option --xml in fc5-convert is meant to turn 5eTools JSON files into XMLs (please correct me if I'm wrong!).

But, when I installed and ran fc5-convert using the instructions in README, I kept running into this message:
`Unknown options: '--xml', '--md', '--background', '--index', '-o'`

If this information helps, I'm on an Intel Mac, and the exact thing I entered to get this message was the template at the end of the README, so
`fc5-convert 5etools --xml --md --background src/main/resources/templates/background2md.txt --index -o dm dm-sources.json ~/git/dnd/5etools-mirror-1.github.io/data wbtw-items.json`

I honestly don't understand what I'm doing, so I would appreciate any help resolving this! (And also any help with the command I need to enter to do what I need to do 😭)

**Comment by ebullient** [comment]
Yes, that should work. 

Can you re-install the CLI? I did just push some updates: 
```
jbang app install --name fc5-convert --force --fresh https://jitpack.io/dev/ebullient/fc5-convert-cli/1.0.0-SNAPSHOT/fc5-convert-cli-1.0.0-SNAPSHOT-runner.jar
```

A) What does the help command show? 

```
fc5-convert 5etools --help
```

B) assuming your homebrew monsters are in a file called `bestiary.json`, I would think this should work, if you just want XML: 

```
fc5-convert 5etools --xml -o somedir bestiary.json
```

The other arguments are only useful if you want markdown, too.

**Comment by isthei** [comment]
After re-installing the CLI, the help command still shows this:
```
Usage: fc5-convert [-hvV] [<input>...] [COMMAND]
      [<input>...]   XML Source file(s)
  -h, --help         Show this help message and exit.
  -v, --verbose      verbose output
  -V, --version      Print version information and exit.
Commands:
  obsidian    Create Obsidian.md Markdown references from XML file
  transform   Transform XML files
  validate    Validate XML files against a schema definition
  completion  bash/zsh completion:  source <(fc5-convert completion)
```
I don't know if this is useful but I also tried steps 1 and 2 of the revised "Starting with 5eTools JSON data" section and ran into the same message: `Unknown options: '--xml', '--md', '--index', '-o'`.

By the way, I really appreciate how quickly you replied!

**Comment by ebullient** [comment]
I figured out why. ;) jitpack had frozen the revisions. I've updated to 1.1.0-SNAPSHOT, and will think about releasing with like.. numbers.. more frequently. =)

Let's see what happens after this:
```
jbang app install --name fc5-convert --force --fresh https://jitpack.io/dev/ebullient/fc5-convert-cli/1.1.0-SNAPSHOT/fc5-convert-cli-1.1.0-SNAPSHOT-runner.jar
```

**Comment by isthei** [comment]
Sorry for the late update. I updated and it now works perfectly for my purposes! Because of this app(?) I can give stats to all my NPCs and run them in Game Master 5 in a political game where my players might get into a fight with anyone at any time. So, this completely saved my game and so much of my time, thank you very much!!

If anyone else who's command-line illiterate needs to do the same thing I did, here is the exact thing I entered:
`fc5-convert 5etools --xml --md --index -o somedir -s Homebrew bestiary.json`
where 'somedir' is the output folder, 'Homebrew' is what's in the 5eTools source field, and 'bestiary.json' is the file name. I have no clue if this is technically correct, but I got the files I needed, so I'm happy.

**Comment by isthei** [comment]
Sorry for the late update. I updated and it now works perfectly for my purposes! Because of this app(?) I can give stats to all my NPCs and run them in Game Master 5 in a political game where my players might get into a fight with anyone at any time. So, this completely saved my game and so much of my time, thank you very much!!

If anyone else who's command-line illiterate needs to do the same thing I did, here is the exact thing I entered:
`fc5-convert 5etools --xml --md --index -o somedir -s Homebrew bestiary.json`
where 'somedir' is the output folder, 'Homebrew' is what's in the 5eTools source field, and 'bestiary.json' is the file name. I have no clue if this is technically correct, but I got the files I needed, so I'm happy.
